lib/node_modules/@stdlib/blas/ext/base/dnansumors/src/main.c

```diff
@@ -0,0 +1,70 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/strided/base/stride2offset.h"
+#include "stdlib/blas/ext/base/dnansumors.h"
```

Comment:
```suggestion
#include "stdlib/blas/ext/base/dnansumors.h"
#include "stdlib/strided/base/stride2offset.h"
```

The package include should always come first.